website/docs/intro/install/examples/verify-cosign.sh

```diff
@@ -4,7 +4,7 @@ IDENTITY="https://github.com/opentofu/opentofu/.github/workflows/release.yml@ref
 cosign \
     verify-blob \
     --certificate-identity "${IDENTITY}" \
-    --signature "tofu_*.sig" \
-    --certificate "tofu_*.pem" \
+    --signature "tofu_"*".sig" \
```

Comment:
Should the quotes be removed all-together?

Reply:
That's also an option :). I can do it if you prefer.

Reply:
yes, we have the `OPENTOFU_VERSION_MAJORMINOR=` variable above but it's missing the patch number for we can't use it directly and we would need to add another variable for this.